### PR TITLE
chore(mssql): Get Microsoft SQL Server ready for release

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

File: plugins/destination/mssql/client/migrate.go [hunk 1]
```diff
@@ -17,16 +17,21 @@ func (c *Client) Migrate(ctx context.Context, tables schema.Tables) error {
...
+		}
+	}()
 
 	if len(table.Columns) == 0 {
 		c.logger.Info().Str("table", table.Name).Msg("Table with no columns, skipping")
 		return nil
 	}
 
-	switch tableExist, err := c.tableExists(ctx, table.Name); {
+	switch tableExist, err := c.tableExists(ctx, table); {
```
Comment (hermanschaaf) [inline plugins/destination/mssql/client/migrate.go:34]: I could be missing something, but it looks like `err` is being shadowed here (and inside the cases), so I don't think the `migrate`-scope `err` that is logged in the defer will ever be non-nil?

Comment (disq) [reply plugins/destination/mssql/client/migrate.go:34]: The `err` in the defer is coming from the named return argument I think.

Comment (hermanschaaf) [reply plugins/destination/mssql/client/migrate.go:34]: Right, but where is it getting set?

Comment (candiduslynx) [reply plugins/destination/mssql/client/migrate.go:34]: yes, the err in defer is exactly from the named ret value

Comment (candiduslynx) [reply plugins/destination/mssql/client/migrate.go:34]: on return

Comment (hermanschaaf) [reply plugins/destination/mssql/client/migrate.go:34]: Oh okay, I didn't know defer worked that way with named return values, TIL

Comment (candiduslynx) [reply plugins/destination/mssql/client/migrate.go:34]: yeah, basically, you can use defer to update named return values, see https://go.dev/blog/defer-panic-and-recover

File: .release-please-manifest.json [hunk 2]
```diff
@@ -80,5 +80,6 @@
   "plugins/source/oracle": "1.0.1",
   "plugins/source/oracle+FILLER": "0.0.0",
   "plugins/source/alicloud": "1.0.0",
-  "plugins/source/alicloud+FILLER": "0.0.0"
+  "plugins/source/alicloud+FILLER": "0.0.0",
+  "plugins/destination/mssql": "1.0.0"
```
Comment (erezrokah) [inline .release-please-manifest.json:84]:
```suggestion
  "plugins/source/alicloud+FILLER": "0.0.0"
```

https://github.com/cloudquery/cloudquery/blob/main/release-please-config.json is the file to update, not this one (this file is used by our bot to determine the next version so putting `1.0.0` here will generate a release PR for `1.0.1`)

**Review by yevgenypats — APPROVED** [review]

**Review by erezrokah — CHANGES_REQUESTED** [review]

File: .release-please-manifest.json (hunk 2 above)
Comment (candiduslynx) [reply .release-please-manifest.json:84]: upd in #6973

**Pull request merged**